src/components/ui/index.ts

```diff
@@ -1,3 +1,4 @@
 export { BottomTabIconLink } from './BottomTabIconLink';
 export { Button, buttonVariants } from './Button';
+export * as Dialog from './Dialog';
```

Comment:
You can write like this instead of using wildcard.

```
export { Body, Content, Footer, Header, Title } from './Dialog';
```

> Too many wildcard imports may lead to possible conflicts when compiling your programs. in addition, because using wildwards implicitly include all other components that are not required by your application, source code tends to be less readable

Reply:
Thanks for your suggestion. I have a concern about it. If we replace it, there may be many conflicts with the names since this is an "index" file. I am currently working on the "Drawer" component, and in fact, I am using the same keywords "Body", "Contents", and so forth.

I was thinking that indexing all the components in this file may bring more problems than benefits. What do you think?

Reply:
As a result of the discussion, it was concluded to rename them with the "Dialog" prefix like "DialogBody". They should all be named-imported/exported in the index.ts.